### PR TITLE
Fix memcached.set slowness

### LINKED ISSUE #14: Order of magnitude increase in cache set time with values of certain size
See Gist: https://gist.github.com/seifertd/8ef466901facae318e20

node-memcached: version 2.1.0
node-cached: version 4.0.0

Cache a string of size ranging from 515,000 bytes to 525,000 bytes in increments of 100 bytes.  Using cached, store time is longer than using plain vanilla memcached (to be expected), but when the object being cached has a size >= 516,000 bytes and size <= 524200, the store time is an order of magnitude higher than the store time of objects outsize that size range.

The plain vanilla memcached library does not suffer from this problem.

Possibly related to: https://github.com/3rd-Eden/node-memcached/pull/205

### PR DESCRIPTION
**Pull request by khoomeister**
Fixes #14

### CODE REVIEW

File: src/cache.coffee [hunk 1]
```diff
@@ -71,7 +71,10 @@ class Cache
     @setDefaults defaults
     @setBackend backend
 
-  applyPrefix: (key) -> "#{@prefix}#{key}"
+  applyPrefix: (key) ->
+    [ @prefix, key].join ''
```
Comment (jkrems) [inline src/cache.coffee]: Space at the end missing..? Looks inconsistent this way.

Comment (khoomeister) [inline src/cache.coffee]: Whoops - fixed!

**Comment by jkrems** [comment]
Once we have confirmation from @seifertd that it fixes the problem - LGTM. Unfortunate but... whatever works I guess..?

**Comment by seifertd** [comment]
@jkrems @khoomeister That works :( See https://gist.github.com/seifertd/8ef466901facae318e20#file-with-pr-15-fixes
:shipit:

**Comment by khoomeister** [comment]
v4.0.1

**Pull request merged**